RushMaverick/ft_transcendence
# Trn 101 create tournament api

Small update to test out tournament

- You can test the tournament by creating 4 users (I had four profiles in chromium) 
- and then go to /tournaments, give tournament a name and then create. 
- Check the tournament id from development console (F12)
- Join that tournament using the id found in dev console with all the four users
- Start and enjoy
